dessia_common/workflow/core.py

```diff
@@ -101,15 +101,44 @@ def dict_to_object(cls, dict_: JsonSerializable, force_generic: bool = False,
     def copy(self, deep: bool = False, memo=None):
         """ Copies and gives a new object with no linked data. """
         return TypedVariable(type_=self.type_, name=self.name)
+    
+    def get_script_elements(self, script: ToScriptElement) -> ToScriptElement:
+        """
+        Update elements of the script representing the workflow.
+
+        return ToSriptElement
+        """
+        types = ["Dict", "Tuple"]
+        try:
+            script.declaration += f", type_={self.type_.__name__}"
+        except AttributeError:
+            if is_typing(self.type_):
+                args = self.type_.__args__
+                if len(args) == 1:
```

Comment:
This is not generic. typings could have any number of args, so we cannot only have two cases for 1 and 2.

You have to forge the str from any number of args

Reply:
it's true, 
I did that because in the case of a dict we don't have just one args,

Reply:
Yes, from my experience, you have these cases : 
- List, InstanceOf, Subclass, Type should have only one arg (exactly 1),
- Dict should have two args (exactly 2),
- Tuple could have any number of args, but at least 1,

You have weird typings on whoch we don't have the hand on. For example if you type this : 
`def function(arg: T = None):`
Then, you have the following type : 
`Optional[T]`
Which have two args, the first one is T, the second one is NoneType. And this is only true when the value is defaulted to None.

We may have more cases, but these are the only one I can think of right now